src/main.tsx

```diff
@@ -1,9 +1,18 @@
-import React from "react";
+/* existing imports */
+import { RouterProvider, createBrowserRouter } from "react-router-dom";
+import Root from "./routes/root";
 import ReactDOM from "react-dom/client";
-import App from "./App.tsx";
+import React from "react";
+
+const router = createBrowserRouter([
+  {
+    path: "/",
+    element: <Root />,
+  },
+]);
 
-ReactDOM.createRoot(document.getElementById("root")!).render(
+ReactDOM.createRoot(document.getElementById("root") as HTMLElement).render(
   <React.StrictMode>
-    <App />
+    <RouterProvider router={router} />
   </React.StrictMode>
 );
```

Comment:
Return the App.tsx files and replace it on line 16 here.
Take your router and put in the APp.tsx file. 
Wrap the router around ThemeProvider and add the <CssBaseline />